ceremonies/src/lib.rs

```diff
@@ -523,28 +523,11 @@ impl<T: Config> Module<T> {
     }
 
     fn find_coprime_below(upper_bound: u64) -> u64 {
```

Comment:
@NicJak i just stumbled upon this. if you always take `n-1` the algorithm becomes predictable. s1 is supposed to be random and coprime, that's the whole point of the algorithm. but it is my bad, i was not explicit about that when i explained it to you, sorry!